frontend/src/metabase/common/hooks/use-entity-list-query/use-entity-list-query.ts

```diff
@@ -17,8 +17,14 @@ export interface UseEntityListOwnProps<TItem, TQuery = never> {
     state: State,
     options: EntityQueryOptions<TQuery>,
   ) => TItem[] | undefined;
-  getLoading: (state: State, options: EntityQueryOptions<TQuery>) => boolean;
-  getLoaded: (state: State, options: EntityQueryOptions<TQuery>) => boolean;
+  getLoading: (
+    state: State,
+    options: EntityQueryOptions<TQuery>,
+  ) => boolean | undefined;
+  getLoaded: (
+    state: State,
+    options: EntityQueryOptions<TQuery>,
+  ) => boolean | undefined;
```

Comment:
I can understand why do we have loading result as not defined, but loaded should be either true or false, don't you think so?

Reply:
It's what we get from the entity framework. The selector returns `boolean | undefined`, without changes to the framework this cannot be changed